ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Debug.scala

```diff
@@ -6,10 +6,10 @@ package com.daml.ledger.participant.state.kvutils
 import java.io.DataOutputStream
 import java.nio.file.{Files, Paths}
 
-import com.daml.ledger.participant.state.kvutils.DamlKvutils._
+import com.daml.ledger.participant.state.kvutils.DamlKvutils.{Envelope => _, _}
```

Comment:
Why hiding `Envelope`?

Reply:
We want `com.daml.ledger.participant.kvutils.Envelope` below. The scoping rules in 2.13 seem to be different which means that we end up with the one from `DamlKvutils` otherwise.